src/Apps/Order/Routes/__tests__/Shipping.jest.tsx

```diff
@@ -254,270 +219,220 @@ describe("Shipping", () => {
+
+        expect(screen.getByRole("combobox")).toHaveValue("US")
+        expect(screen.getByRole("combobox")).toBeDisabled()
       })
-      const page = new ShippingTestPage(wrapper)
 
-      fillAddressForm(page.root, validAddress)
+      it("sets and enables country select if artwork only ships domestically and is in the EU", async () => {
+        const domesticShippingEUOrder = cloneDeep(order) as any
+        const artwork = domesticShippingEUOrder.lineItems.edges[0].node.artwork
 
-      expect(mockCommitMutation).not.toHaveBeenCalled()
+        Object.assign(artwork, {
+          onlyShipsDomestically: true,
+          euShippingOrigin: true,
+          shippingCountry: "IT",
+        })
 
-      page.find(`[data-test="save-address-checkbox"]`).first().simulate("click")
+        renderWithRelay({
+          CommerceOrder: () => domesticShippingEUOrder,
+          Me: () => meWithoutAddress,
+        })
 
-      await page.clickSubmit()
+        expect(screen.getByRole("combobox")).toHaveValue("IT")
+        expect(screen.getByRole("combobox")).toBeEnabled()
+      })
 
-      expect(mockCommitMutation).toHaveBeenCalledTimes(1)
-      expect(mockCommitMutation).toHaveBeenCalledWith(
-        expect.objectContaining({
-          variables: {
-            input: {
-              id: "1234",
-              fulfillmentType: "SHIP",
-              phoneNumber: validAddress.phoneNumber,
-              shipping: {
-                ...validAddress,
-                phoneNumber: "",
-              },
-            },
-          },
+      it("sets shipping on order and saves address on user", async () => {
+        mockCommitMutation.mockResolvedValueOnce(settingOrderShipmentSuccess)
+        renderWithRelay({
+          CommerceOrder: () => order,
+          Me: () => meWithoutAddress,
         })
-      )
-    })
 
-    it("commits the mutation with shipping option", async () => {
-      mockCommitMutation.mockResolvedValueOnce(settingOrderShipmentSuccess)
-      const wrapper = getWrapper({
-        CommerceOrder: () => testOrder,
-        Me: () => emptyTestMe,
-      })
-      const page = new ShippingTestPage(wrapper)
+        fillAddressFormTL(validAddress)
+        await saveAndContinue()
 
-      const address = {
-        ...validAddress,
-        region: "New Brunswick",
-        country: "US",
-      }
-      fillAddressForm(page.root, address)
-
-      await page.clickSubmit()
-      expect(mockCommitMutation).toHaveBeenCalledTimes(2)
-      expect(mockCommitMutation).toHaveBeenNthCalledWith(
-        1,
-        expect.objectContaining({
-          variables: {
-            input: {
-              id: "1234",
-              fulfillmentType: "SHIP",
-              phoneNumber: address.phoneNumber,
-              shipping: {
-                ...address,
-                phoneNumber: "",
-              },
+        expect(mockCommitMutation).toHaveBeenCalledTimes(2)
+
+        let mutationArg = mockCommitMutation.mock.calls[0][0]
```

Comment:
Previously we only checked the mutation variables, which can be confusing sometimes. The new pattern here uses the [`mock` property](https://jestjs.io/docs/mock-functions#mock-property), so we can more explicitly check both the mutation name and the variables.